language-json.md

```diff
@@ -0,0 +1,49 @@
+# JavaScript Object Notation
+
+AJAX response have to follow [RPC convention](http://docs.juwai.io/rpc/property.html#module-jw.property.views.rpc).
```

Comment:
ok, I'll remove it, I tested this link before with different IP addresses, I thought it is accessible for all public IP addresses, so I linked it in the documentation.